tests/text/test_pipeline_with_custom_head.py

```diff
@@ -1,23 +1,37 @@
 import os
+from pathlib import Path
 from tempfile import mkdtemp
 
-from biome.text import Pipeline, PipelineConfiguration
-from biome.text.configuration import (
-    FeaturesConfiguration,
-    VocabularyConfiguration,
-)
-from biome.text.data import DataSource
-from biome.text.modules.heads import TaskHeadConfiguration, TextClassification
-from tests.test_context import TEST_RESOURCES
+import pytest
 
+from biome.text import Pipeline, PipelineConfiguration, Dataset
+from biome.text.configuration import FeaturesConfiguration,VocabularyConfiguration
+from biome.text.modules.heads import TaskHeadConfiguration, TextClassification
 
 class MyCustomHead(TextClassification):
     """Just a head renaming the original TextClassification head"""
-
     pass
 
+@pytest.fixture
+def training_dataset() -> Dataset:
+    """Creates the training dataset and gives the structure"""
+    resources_path = Path(__file__).parent.parent.parent / "tests" / "resources" / "data"
```

Comment:
We could leave a TODO here:
```suggestion
    # TODO: We must cleanup the `resources` and should use a fixture to point to them
    resources_path = Path(__file__).parent.parent.parent / "tests" / "resources" / "data"
```

Reply:
Yeah, I'll add it. We must standarize this procedure.